src/navigation-bar.tsx

```diff
@@ -474,6 +369,10 @@ export const FunnelNavigationBar: React.FC<FunnelNavigationBarProps> = ({
     isInverse = false,
     topFixed = true,
 }) => {
+    const {isTabletOrSmaller} = useScreenSize();
+
+    logo = logo ?? <Logo size={isTabletOrSmaller ? 40 : 48} />;
```

Comment:
we were talking me and @yceballost about the fact that if you don't pass the logo, the default size would be different. I've left like this for not breaking things, but we'd better decide if we're ok with this behavior:

- if a `Logo` is explicitly passed without any explicit sizing, then the size would be `48` irrespective of the device. of cource the parent can add the logics like the one above, but it should be explicit
- if a `Logo` is not explicitly passed, the default one is used with the sizing described above

You can play around with this, by commenting out the logo, here: https://bit.ly/3ozCvCM

are we ok with this?

Reply:
Yes, it's fine